lib/Ohmbrewer_Thermostat.cpp

```diff
@@ -1,93 +1,99 @@
 #include "Ohmbrewer_Thermostat.h"
 #include "Ohmbrewer_Screen.h"
 #include "Ohmbrewer_Publisher.h"
+#include "pid.h"
 
+//Define Variables we'll be connecting to with PID
+double setPoint;
+double input;
+double output;
 
+//Define the aggressive and conservative Tuning Parameters for PID
+double aggKp=4;
+double aggKi=0.2;
+double aggKd=1;
+double consKp=1;
+double consKi=0.05;
+double consKd=0.25;
 
-/**
- * The desired target temperature. Defaults to Celsius
- * @returns The target temperature in Celsius, as a Temperature object pointer
- */
-Ohmbrewer::Temperature* Ohmbrewer::Thermostat::getTargetTemp() const {
-    return _targetTemp;
-}
-
-/**
- * Sets the target temperature
- * @param targetTemp The new target temperature in Celsius
- * @returns The time taken to run the method
- */
-const int Ohmbrewer::Thermostat::setTargetTemp(const double targetTemp) {
-    unsigned long start = millis();
-
-    _targetTemp->set(targetTemp);
+//PID windowing variables
+int windowSize = 5000;              //changeable value
```

Comment:
Since the values for these will be particular to a given Thermostat, all of these PID variables should be provided as members of Thermostat, with getters and setters. You can still set default values, but it would happen in Thermostat.h.

I'm assuming this stuff is here for now for debugging, though, right? :wink: